app/views/bookmarks/show.html.erb

```diff
@@ -1,11 +1,22 @@
 <div class="row">
   <div class="col-md-9">
-    <h4><a href= <%= @url %> > <%= @url %> </a> </h4>
+    <h4><a target="_blank" href= <%= @url %> > <%= @url %> </a><h4>
+    <p><%= embed_description(@url) %></p>
+
+    <% if find_type(@url) == 'video' %>
+      <%= embed_media(@url, 800) %>
+    <% else %>
+      <div class="img">
+        <a target="_blank" href=<%=@url%>> <img src= <%= embed_media(@url, 500) %> width="400" height="400"></a>
```

Comment:
`embed_media` on line 7/10 triggers 3rd HTTP request to embedly's servers. All 3 requests return identical data from embedly.